### ISSUE TITLE
R2SCAN-3c

### ISSUE THREAD

**Issue by kuitpo** [pyscf/pyscf#825]
Feature request for R2SCAN-3c. The Summary and Conclusions of the paper makes the case pretty well.
[](https://doi.org/10.1063/5.0040021)
Requires D4, gCP and a tailored basis set.
R2SCAN is used unaltered, and I see there is a pull request for libxc 5.1.0 which contains that DFA.

**Comment by awvwgk** [comment]
Feel free to ping me in case there are any questions regarding usage or integration of [dftd4](https://github.com/dftd4/dftd4/tree/main/python) or [gcp](https://github.com/grimme-lab/gcp). The mTZVPP basis set will enter the basis set exchange at some point (see https://github.com/MolSSI-BSE/basis_set_exchange/issues/152).

**Comment by mkrompiec** [comment]
@hebrewsnabla are there still plans to implement b97-3c?

**Comment by hebrewsnabla** [comment]
> @hebrewsnabla are there still plans to implement b97-3c?

Yes. I will continue the progress of #1358 in pyscf-forge. But, still, it will be much more comfortable if we can have a python api for gcp, which may be coming soon according to https://github.com/grimme-lab/gcp/issues/5.

**Comment by awvwgk** [comment]
> But, still, it will be much more comfortable if we can have a python api for gcp, which may be coming soon according to https://github.com/grimme-lab/gcp/issues/5.

The gcp development currently has pretty much halted as far as I can tell, maybe it is worth to reach out to the people in the Grimme group to get this project of the ground again.
